aiida_cusp/calculators/calculation_base.py

```diff
@@ -277,13 +286,32 @@ def restart_copy_remote(self, folder, calcinfo):
         remote_comp_uuid = remote_data.computer.uuid
         exclude_files = self.restart_files_exclude()
         overwrite_poscar = self.inputs.restart.get('contcar_to_poscar', True)
+
+        jobs = dict(self.inputs.custodian.get('jobs', {}))
+        suffixes = [""]
+        for job in jobs:
+            suffixes.append(job.get("suffix", ""))
+        suffixes = list(set(suffixes))
+
         for name, abspath, relpath in self.remote_filelist(remote_data):
-            if name in exclude_files:
+
+            # detect suffix and take it out
+            has_suffix = False
+            current_suffix = ""
+            for suffix in suffixes:
+                if name.endswith(suffix):
+                    has_suffix = True
+                    current_suffix = f"{suffix}"
+            if has_suffix:
+                bare_name = name[:-len(current_suffix)]
+
+            if bare_name in exclude_files:
                 continue
```

Comment:
If we do not have a suffix (i.e. `jobs` is an empty `dict`) `bare_name` remains undefined and we never hit the continue (I guess that's the reason why the tests are failing :) )

Reply:
ah. you are right. 🤦‍♀️